nym-connect/src/components/Growth/TestAndEarnButtonArea.tsx

```diff
@@ -25,7 +25,7 @@ export const Wrapper: FCWithChildren<{ disabled: boolean }> = ({ disabled, child
 export const TestAndEarnButtonArea: FCWithChildren = () => {
   const clientContext = useClientContext();
   const context = useTestAndEarnContext();
-  const disabled = clientContext.connectionStatus !== ConnectionStatusKind.connected;
+  const disabled = clientContext.connectionStatus !== 'connected';
```

Comment:
I'm less keen on the string union return types and prefer enums

Reply:
changed to enums